experiments/HSA_20150714_MI/hsa_MI_20150714.py

```diff
@@ -0,0 +1,291 @@
+#! /usr/bin/env python
+"""
+Script for generation of input files for Aspirin and Naproxen binding to HSA by ITC.
+"""
+
+from itctools.itctools import ureg, Quantity
+from itctools.procedures import ITCProtocol, ITCExperimentSet, ITCExperiment, ITCHeuristicExperiment
+from itctools.materials import Solvent, Compound, SimpleSolution
+from itctools.labware import Labware, PipettingLocation
+import pprint
+
+
+## Is this volume correct? I can try adding units.
+
+# The sample cell volume in microliters
+cell_volume = 202.8
```

Comment:
Actually, its because the protocol volumes are unitless, and it is used only there. Add units to both, and it should be okay. I am not sure about the printed .inj file though. That one might not work with units. I'll open an issue to fix this.